src/bci_build/package.py

```diff
@@ -1052,10 +1055,18 @@ def uid(self) -> str:
     @property
     def build_tags(self) -> List[str]:
         tags = []
+        # deconflict tags by the stability ordering
+        _stability_tag_ordering = (None, "stable", "oldstable")
+        relsuffix = "%RELEASE%"
+        if self.stability_tag and self.stability_tag in _stability_tag_ordering:
+            relsuffix = f"{_stability_tag_ordering.index(self.stability_tag)}.%RELEASE%"
         for name in [self.name] + self.additional_names:
-            for ver_label in [self.version_label] + self.additional_versions:
+            ver_labels = [self.version_label]
```

Comment:
Wouldn't this be a simpler solution?
```suggestion
            for ver_label in self.additional_versions + [self.version_label]:
```